netket/vqs/mc/mc_state/state.py

```diff
@@ -549,6 +550,50 @@ def log_value(self, σ: jnp.ndarray) -> jnp.ndarray:
         """
         return jit_evaluate(self._apply_fun, self.variables, σ)
 
+    def local_estimators(self, op: AbstractOperator, *, chunk_size: int = None):
```

Comment:
Why do we support specifying chunk_size here? 
In general we don't support it elsewhere so I would not support it here either?

Reply:
As we discussed on Slack, I think being able to specify `chunk_size` here is useful, as the necessary `chunk_size` can vary between operators for a fixed `MCState`.  I know this argument also applies to `expect` etc., but let's do one thing at a time.